### ISSUE TITLE
Remove `@timestamp` field from below `fields` in `states-inventory-hardware`

### ISSUE THREAD

**Issue by f-galland** [wazuh/wazuh-indexer-plugins#172]
## Description

It was reported that the `@timestamp` field is being indexed under a `fields` top level object. This is not according to specs and it should be removed.

**Comment by f-galland** [comment]
![Image](https://github.com/user-attachments/assets/a10bcaca-0b02-483e-8a03-0530a8930274)

When taking a closer look, we can see the `fields` object is not nested within `_source`, but right at the top level.
This is actually not part of the document itself, but gets added when [running a query with `stored_fields`](https://opensearch.org/docs/latest/search-plugins/searching-data/retrieve-specific-fields/#searching-with-stored_fields).

In short, `fields` is not part of the indexed document.


Reference links:
* https://opensearch.org/docs/latest/api-reference/document-apis/get-documents/#response-body-fields
